src/pages/workspaces/List.js

```diff
@@ -68,12 +68,12 @@ const styles = {
   },
   longTitle: {
     ...Style.elements.card.title,
-    whiteSpace: 'nowrap', textOverflow: 'ellipsis', flex: 1
+    ...Style.noWrapEllipsis, flex: 1
```

Comment:
I wonder why this worked before without `overflow: 'hidden'`.

Reply:
because `overflow: 'hidden'` is in `Style.elements.card.title`